src/main/kotlin/org/wycliffeassociates/otter/common/domain/ImportResourceContainer.kt

```diff
@@ -143,19 +127,33 @@ class ImportResourceContainer(
     }
 
     private fun getCategories(rc: ResourceContainer): List<Collection> {
-        return arrayListOf(
-                Collection(1, "bible-ot", "bible-ot", "Old Testament", null),
-                Collection(2, "bible-nt", "bible-nt", "New Testament", null)
-        )
+        val collections = arrayListOf<Collection>()
+        val config = rc.config
+        config.let {
+            if (config is OtterResourceContainerConfig) {
+                config.extendedDublinCore?.let {
+                    for (cat in it.categories) {
+                        collections.add(
+                                Collection(cat.sort, cat.identifier, cat.identifier, cat.title, null)
+                        )
+                    }
+                }
+            }
+        }
+        return collections
     }
 
     private fun getProjectsInCategory(manifest: Manifest, categorySlug: String): List<Project> {
         val projects = manifest.projects.filter { it.categories.contains(categorySlug) }
         return projects
     }
 
-    private fun getProjectsWithoutCategory(manifest: Manifest): List<Project> {
-        val projects = manifest.projects.filter { it.categories.isEmpty() }
+    private fun getProjectsNotInCategories(manifest: Manifest, categorySlugs: List<String>): List<Project> {
+        val projects = manifest.projects.filter {
+            categorySlugs.isEmpty() // no categories were added (maybe config is missing)
+                || it.categories.isEmpty() // project doesn't belong to a category
+                || it.categories.map { categorySlugs.contains(it) }.none() // some categories are in config, not all
```

Comment:
Should this be `filter { !categorySlugs.contains(it) }.isNotEmpty()`? 

`it.categories.map { categorySlugs.contains(it) }.none()` would return the same as `it.categories.none()`
`it.categories.filter { categorySlugs.contains(it) }.none()` would be true if none of the categories are in the config, which I don't think is quite what we want. We want the expression to be true if some of the categories are not in config, right?

Reply:
Let's say I have a project with categories `[ "bible-ot", "other" ]`. The config defines `bible-ot` but not `other`. In this scenario, we want to import the project under only `bible-ot`, right? But if the project category was just `[ "other" ]`, we'd want to import as a project not in a category. If that's correct, then the solution in your comment looks good 👍. (Just trying to make sure I picked up on the right end goal)

Reply:
There's two functions, getProjectsInCategory is going to be called on all categories that came from the rc config (in this case, it handles bible-ot)

getProjectsNotInCategories should pull everything else, 
so, "genesis" being in "bible-ot", will have an intersection, and not be included in projects without a category

"matthew" being in "other" will not have an intersection, and will get included

This means that Matthew gets placed at the same level as Old Testament, but assuming the wizard only checks what do to on selection, this is fine- it means all projects are accounted for, even though the hierarchies are now a little weird